nengo/processes.py

```diff
@@ -299,7 +299,7 @@ def coerce(self, instance, data):
             # make sure this is the same size as previous items
             if size != size_out and size_out is not None:
                 raise ValidationError(
-                    "time %g has size %d instead of %d" % (time, size, size_out),
+                    "time %g has size %d instead of %s" % (time, size, size_out),
```

Comment:
Why did this change?

Reply:
The warning you get is `Argument 'builtins.NoneType' does not match format type 'd' (bad-string-format-type)` because pylint doesn't look at the if block context, at least in this case. I'd stick with `%s` though because it's going to give you the same thing as `%d`.